frontend/src/components/shared/StreamlitMarkdown/RemarkColoredText.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// @ts-nocheck
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
nit: I think it would be better to explicitly add those dependencies to our package.json instead of ignoring this warning.

Reply:
Right 👍 They were in `node_modules`, because other packages required them, but we do not have such guarantee, now they are added explicitly into `package.json` and `yarn.lock` 👍